src/Illuminate/Support/Collection.php

```diff
@@ -1266,6 +1266,25 @@ public function toBase()
         return new self($this);
     }
 
+    public function __get($name)
+    {
+        $nameToMethod = [
+            'where' => 'filter',
+            'unless' => 'reject',
+            'do' => 'each',
+            'extract' => 'map',
+            'sum' => 'sum',
+            'inOrderOf' => 'sortBy',
+            'inReverseOrderOf' => 'sortByDesc',
+        ];
```

Comment:
With that array we can't make the `$invoices->each->pay();` announced in the [LN article ](https://laravel-news.com/2016/11/higher-order-messaging). Can we ? We have to do `$invoices->do->pay();` instead. Maybe am I missing something somewhere else ? In this case, I think we should use the original methods too.
Thx.